### PR TITLE
Apply feedback after testnet migration test - Closes #1082

### LINKED ISSUE #1082: Improve migration guide
The Core v3 migration guide needs to be improved in the following points:

- Intro section should say clearly what the lisk migrator script is doing.
- Intro section should sum up quickly how the migration process works.
- lisk-migrator should not be renamed, so that the checksum can be verified correctly.
- add note to run lisk-migrator with screen in the background
- add note that Ubuntu 18 and 20 versions  are supported

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1082

### CODE REVIEW

File: modules/ROOT/pages/management/migration.adoc [hunk 1]
```diff
@@ -13,7 +13,11 @@
 :url_core_2_update: master@lisk-core::update/application.adoc
 :url_sdk_protocol_voteweight: master@lisk-sdk:protocol:consensus-algorithm.adoc#voting_and_weight
 
-How to migrate a Lisk Core 2.1.6 node to Lisk Core 3.0.0.
+This guide explains how to migrate a Lisk Core 2.1.6 node to Lisk Core 3.0.0 with the Lisk migrator.
```
Comment (ManuGowda) [inline modules/ROOT/pages/management/migration.adoc]:
```suggestion
This guide explains how to migrate a Lisk Core 2.1.6 node to Lisk Core 3.0.0 using the Lisk migrator.
```

File: modules/ROOT/pages/management/migration.adoc [hunk 2]
```diff
@@ -13,7 +13,11 @@
...
+
+The Lisk migrator script will generate a new genesis block for Lisk Core 3.0.0.
```
Comment (ManuGowda) [inline modules/ROOT/pages/management/migration.adoc]:
```suggestion
The Lisk migrator CLI will generate a new genesis block for Lisk Core 3.0.0.
```

File: modules/ROOT/pages/management/migration.adoc [hunk 3]
```diff
@@ -287,6 +299,44 @@ The absolute path to the directory, where the newly generated genesis block shou
...
+
+With `screen` you can detach the current terminal window into the background:
+
+.Example (Mainnet)
+[source,bash]
+----
+screen -dmSL migration lisk-migrator --snapshot-height ${snapshotHeight} --output ~/.lisk/lisk-core/config/mainnet/genesis_block.json --lisk-core-path ~/lisk-main
+----
+
+And then short before the migration you may reattach to it to check if everything is working fine.
```
Comment (ManuGowda) [inline modules/ROOT/pages/management/migration.adoc:317]: short?

File: modules/ROOT/pages/management/migration.adoc [hunk 4]
```diff
@@ -28,13 +32,19 @@ In this case, the following steps 1.4, 2.7, 2.8, and 2.9 in this guide can be sk
 :sectnumlevels: 5
 == Preparation
 
+.System requirements
```
Comment (ManuGowda) [inline modules/ROOT/pages/management/migration.adoc:35]: what about node.js version? 12.22.1? someone mentioned in discord

File: modules/ROOT/pages/management/migration.adoc (hunk 3 above)
Comment (Tschakki) [reply modules/ROOT/pages/management/migration.adoc:317]: yeah, like a few minutes before

File: modules/ROOT/pages/management/migration.adoc (hunk 4 above)
Comment (Tschakki) [reply modules/ROOT/pages/management/migration.adoc:35]: Are you sure about this?  At least I did run the lisk-migrator on my Ubuntu without having node installed before.

Comment (ManuGowda) [reply modules/ROOT/pages/management/migration.adoc:35]: true, but someone in the community had some nodejs version installed and mentioned about an issue with that, but i am not sure what will happen if have lower nodejs version and run migration, just to be on safer side we can say, `if you have node.js installed ensure to use 12.22.1` something like this?

File: modules/ROOT/pages/management/migration.adoc (hunk 3 above)
Comment (ManuGowda) [reply modules/ROOT/pages/management/migration.adoc:317]: ah ok

**Review by ManuGowda — DISMISSED** [review]

File: modules/ROOT/pages/management/migration.adoc (hunk 4 above)
Comment (Tschakki) [reply modules/ROOT/pages/management/migration.adoc:35]: Ok, I added a note now

**Review by ManuGowda — APPROVED** [review]

**Pull request merged**